Kenneth-Macharia/fastfoodfast_flaskAPI
# #160854978 Code linting amendments

What does this PR do?
Amends code to meet pylint stds

Description of Task to be completed?
Ensure a min pylint score of 8.0

How should this be manually tested?
Clone the repo, CD into repo folder, activate 'env' virtual environment, run 'pylint ./app' on the command prompt. Output should be as shown on screenshot below.

What are the relevant pivotal tracker stories?
#160854978

![screen shot 2018-09-28 at 8 48 17 pm](https://user-images.githubusercontent.com/42747960/46224743-e53ba180-c35f-11e8-91cd-3214f82ffee8.png)
